nvi-commons/src/test/java/no/sikt/nva/nvi/common/service/PublicationLoaderServiceTest.java

```diff
@@ -41,46 +40,17 @@ void setUp() {
   }
 
   @ParameterizedTest
-  @MethodSource(EXAMPLE_PROVIDER)
+  @MethodSource("exampleDocumentTestProvider")
   void shouldNotFailWhenValidatingExampleDocument(String filename) {
     var actual = parseExampleDocument(filename);
     assertThatNoException().isThrownBy(actual::validate);
   }
 
   @ParameterizedTest
-  @MethodSource(EXAMPLE_PROVIDER)
-  void shouldGetExpectedFieldsFromExampleDocument(String filename, PublicationDto expected) {
+  @MethodSource("exampleDocumentTestProvider")
+  void shouldGetExpectedDataFromExampleDocuments(String filename, PublicationDto expected) {
     var actual = parseExampleDocument(filename);
-    assertThat(actual)
-        .usingRecursiveComparison()
-        .ignoringFields("publicationChannels", "contributors", "topLevelOrganizations")
-        .isEqualTo(expected);
-  }
-
-  @ParameterizedTest
-  @MethodSource(EXAMPLE_PROVIDER)
-  void shouldGetExpectedContributorsFromExampleDocument(String filename, PublicationDto expected) {
-    var expectedContributors = expected.contributors();
-    var actualContributors = parseExampleDocument(filename).contributors();
-    assertThat(actualContributors).containsExactlyInAnyOrderElementsOf(expectedContributors);
-  }
-
-  @ParameterizedTest
-  @MethodSource(EXAMPLE_PROVIDER)
-  void shouldGetExpectedPublicationChannelsFromExampleDocument(
-      String filename, PublicationDto expected) {
-    var expectedChannels = expected.publicationChannels();
-    var actualChannels = parseExampleDocument(filename).publicationChannels();
-    assertThat(actualChannels).containsExactlyInAnyOrderElementsOf(expectedChannels);
-  }
-
-  @ParameterizedTest
-  @MethodSource(EXAMPLE_PROVIDER)
-  void shouldGetExpectedTopLevelOrganizationsFromExampleDocument(
-      String filename, PublicationDto expected) {
-    var expectedOrganizations = expected.topLevelOrganizations();
-    var actualOrganizations = parseExampleDocument(filename).topLevelOrganizations();
-    assertThat(actualOrganizations).containsExactlyInAnyOrderElementsOf(expectedOrganizations);
+    assertThat(actual).usingRecursiveComparison().ignoringCollectionOrder().isEqualTo(expected);
```

Comment:
These are the magic words, remember them well.

Reply:
Curiously, this _mystical invocation_ in the _ancient language_ is easy to understand.